scripts/settings_template.tex

```diff
@@ -0,0 +1,114 @@
+\documentclass{article}
+
+\usepackage{swiftnav}
+
+\usepackage{endnotes}
+
+\usepackage{standalone}
+\usepackage{longtable}
+\usepackage{register}
+\usepackage{bytefield}
+\usepackage{booktabs}
+\usepackage{minibox}
+\usepackage{float}
+\usepackage{amsmath}
+\usepackage{caption}
+
+\setlength{\regWidth}{0.4\textwidth}
+
+\floatstyle{plain}
+\newfloat{field}{h}{fld}
+\floatname{field}{Field}
+
+\numberwithin{table}{subsection}
+\numberwithin{field}{subsection}
+
+\usepackage{draftwatermark}
+\SetWatermarkLightness{0.9}
+\SetWatermarkScale{1}
+
+\renewcommand{\regLabelFamily}{}
+
+% ---------------------------------------------------------------------------
+
+\version{1.0}
+\title{Piksi Settings}
+\mysubtitle{Piksi Firmware version  v\theversion}
```

Comment:
Still think it would be good to pull this from git. Also I think the version number should be v0.14 or v0.15 until we decide to ship v1.0. Don't want people getting too excited!